keyboards/kbdfans/odin75/odin75.c

```diff
@@ -0,0 +1,41 @@
+ /* Copyright 2022 DZTECH <moyi4681@live.cn>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+#include "lib/bongocat.h"
+#ifdef OLED_ENABLE
+  oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+  //return OLED_ROTATION_180;
+  return OLED_ROTATION_0;
+}
+    bool oled_task_kb(void) {
+        if (!oled_task_user()) { return false; }
+        if (!oled_task_user()) { return false; }
```

Comment:
Looks like you added this twice?  
```suggestion
        if (!oled_task_user()) { return false; }
```